community-spotlight-conference-backend-world-affairs-conference.mdx

```diff
@@ -0,0 +1,92 @@
+
+![Attendee administrative table](/images/cs-wac-admin-table.png)
+
+In order to perform interactive analytics on the data itself, the team relied on [Xata aggregations](https://xata.io/docs/sdk/aggregate). This allowed them to chart the number of sign ups over time and segment based on schools and other profile attributes. The view of the data will help future conference administrators with outreach, promotion and further optimization.
+
+![Dashboard for attendee and speaker analytics](/images/cs-wac-admin-dashboard.png)
+
+Inline editing for the website was a really important feature in order to support real time modifications without providing full access to the frontend or administrative dashboard. Content administrators were able to make changes through this admin view, save and then automatically publish to the website. This was all handled through [Xata transactions](https://xata.io/docs/sdk/transaction). This CMS style editing was also used for user and speaker profiles, storing images with [file attachments](https://xata.io/docs/sdk/file-attachments). Having a single database that supported text, relationships and images was a really important part of the decision process when defining this tech stack to support multiple use cases.
+
+![Administrative CMS for website team](/images/cs-wac-admin-cms.png)
+
+Both the conference and administrative dashboard can be found in their [open source GitHub repository](https://github.com/worldaffairsconference/worldaffairscon.org).
+
+# Lucia + Xata
+
+One of the more critical aspects of the application was authentication. The original auth solution chosen for by the modernization team for the main website was Next Auth, but there were a number of issues with it. At first, a magic link was used for registration but because some schools have very strict email filters, they had to switch to an auth based login.
+
+Eventually, [Lucia Auth](https://lucia-auth.com/) was chosen as the solution. The pairing of Xata and Lucia ended up being a perfect fit. In just 100 lines of code, Samuel and the team had auth working with full type safety for passing metadata back to Xata. It was really important that they were able to add custom metadata to records. Lucia gives you all the core parts of an auth system, but it’s really up to you on how you integrate it and it worked really well with Xata and the rest of the tech stack. Samuel plans to eventually split out the Xata-Lucia integration into it’s own open source tool.
```

Comment:
```suggestion
Eventually, [Lucia Auth](https://lucia-auth.com/) was chosen as the solution. The pairing of Xata and Lucia ended up being a perfect fit. In just 100 lines of code, Samuel and the team had auth working with full type safety for passing metadata back to Xata. It was really important that they were able to add custom metadata to records. Lucia gives you all the core parts of an auth system, but it’s really up to you on how you integrate it and it worked really well with Xata and the rest of the tech stack. 
```

Reply:
Suggesting to remove this as it mentions OSS

Reply:
Agree. It is still in my plans to release it, as I have recently upgraded to match the latest version of Lucia, but don't want to commit to anything